engine/src/flutter/impeller/typographer/text_frame.cc

```diff
@@ -146,6 +146,17 @@ bool TextFrame::IsFrameComplete() const {
   return bound_values_.size() == run_size;
 }
 
+Font TextFrame::GetFont() const {
```

Comment:
Should this return a `const Font&`?  I don't know how heavy those are.

Reply:
Done